plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/guru/VMwareGuru.java

```diff
@@ -686,8 +400,7 @@ public int compare(NicTO arg0, NicTO arg1) {
         return new Pair<Boolean, Long>(Boolean.FALSE, new Long(hostId));
     }
 
-    @Override
-    public boolean trackVmHostChange() {
+    @Override public boolean trackVmHostChange() {
```

Comment:
Just format stuff: these annotations have been moved to the same line as the method's firm all over the class